scripts/apps/authoring/views/article-edit.html

```diff
@@ -520,7 +521,8 @@
         data-item="item.extra[field._id]" data-html="true" data-limit="schema[field._id].maxlength"></span>
     <div sd-editor3
         data-single-line="field.field_options.single"
-        data-value="item.extra[field._id]"
```

Comment:
this didn't work for assignment?

Reply:
The problem was that on angular on value was not updated with the last html from editor3. So I sent the item and the field name and the html is initialized/saved from/to item.